committee-steering/governance/sig-governance.md

```diff
@@ -1,62 +1,135 @@
+  and Subproject Owners.
+
+#### Requirements
+- Members *MUST* be [community members] to be eligible to hold a leadership role
+  within a SIG.
+- SIGs *MAY* prefer various levels of domain knowledge depending on the
+role. This should be documented.  
+- People management interests - there's a lot of us!    
+
+  #### Escalations
+- Membership disagreements *MAY* be escalated to the SIG Chairs. SIG Chair
+membership disagreements may be escalated to the Steering Committee.
+
+  #### New members and Stepping Down
+- Members *MAY* decide to step down at anytime and propose a replacement.  Use
+lazy consensus amongst other members with fallback on majority vote to accept
+proposal.  The candidate *SHOULD* be supported by a majority of SIG Members or
+Subproject Contributors (as applicable).
+- Members *MAY* select additional members through a [super-majority] vote
+amongst members. This *SHOULD* be supported by a majority of SIG Members or
+Subproject Contributors (as applicable).
 
 ### Chair
 
 - Chair
-  - Run operations and processes governing the SIG
-  - Number: 2-3
-  - Membership tracked in [sigs.yaml]
-
-### Tech Lead
+If no tech lead role is present, Chair assumes responsibilities from #tech-lead
+section in addition
+- Number: 2-3
+- Membership tracked in [sigs.yaml]  
+
+Run operations and processes governing the SIG:
+  - *SHOULD* define how priorities and commitments are managed and delegate to
+  other members as needed  
+  - *SHOULD* drive charter changes, including creation, to get community buy-in
+  but *MAY* delegate content creation to community members
+  - *MUST* identify, track, and maintain the groups features for current release
+  and serve as point of contact for the release team  
+  - *MAY* delegate the creation of a SIG roadmap to other members
+  - *MUST* organize a main group meeting and make sure sigs.yaml is up to date
+  including subprojects and their meeting information but *SHOULD* delegate the
+  need for subproject meetings to owners  
+  - *SHOULD* facilitate meetings but *MAY* delegate to other members or future
+  chairs/chairs in training
+  - *MUST* Create and maintain a CONTRIBUTING.md document in the appropriate SIG
+  folder. *MAY* ask for help from community members to update / send in PRs.
+  - *MUST* Organize KubeCon Intros and Deep Dives with CNCF Event staff and
+  approve presented content but *MAY* delegate to others to create material and
+  present  
+  - *MUST* Ensure meetings are recorded and made available
+  - *MUST* Report activity with the community via k-dev mailing list at least
+  once a quarter (slides, video from kubecon, etc)
+  - *MUST* Coordinate sponsored working group updates to the SIG and the wider
+  community  
+ - *MUST* coordinate communication and be a connector with other community
+ groups like SIGs and the Steering Committee but may delegate the actual
+ communication and creation of to other community members where appropriate  
```

Comment:
creation of what?

Reply:
oops forgot the word content

...and creation of content to other...